security/docker/Dockerfile.node-agent-k8s

```diff
@@ -9,5 +9,7 @@ FROM gcr.io/distroless/static as distroless
 
 # This will build the final image based on either default or distroless from above
 FROM ${BASE_DISTRIBUTION}
-ADD node_agent_k8s /
+COPY node_agent_k8s /
+RUN apt-get update && apt-get install -y --no-install-recommends ca-certificates && update-ca-certificates && apt-get clean  && rm -rf /var/lib/apt/lists/*
+RUN chmod 755 /node_agent_k8s
```

Comment:
I think this was just remved for distroless as this will break distroless. Is this required? It may be -- see https://github.com/istio/istio/issues/14718.

Reply:
I think it is required for the executable permission.

Reply:
chmod line is removed since distroless does not have chmod.

Reply:
if nodea_agent_k8s needs to be `0755` please use chmod in the makefiles.

Reply:
To be consistent with the Dockerfile of other executable, e.g., Pilot, https://github.com/istio/istio/blob/master/pilot/docker/Dockerfile.pilot, https://github.com/istio/istio/blob/master/tools/istio-docker.mk#L144, no chmod line is added.